src/precice/impl/SolverInterfaceImpl.cpp

```diff
@@ -88,6 +89,7 @@ void SolverInterfaceImpl:: configure
   xml::configure(config.getXMLTag(), configurationFileName);
   if(_accessorProcessRank==0){
     INFO("Configuring preCICE with configuration: \"" << configurationFileName << "\"" );
+    INFO("This is preCICE version " << PRECICE_VERSION);
```

Comment:
I would definitely put the "This is preCICE" line before "Configuring...".

Also, I think the startup time is useful, what was the reason for removing it?

Reply:
The time is written by the logger anyway, right?
```
(0) 16:36:09 [impl::SolverInterfaceImpl]:91 in configure: Configuring preCICE with configuration: "../precice-config.xml"
(0) 16:36:09 [impl::SolverInterfaceImpl]:92 in configure: This is preCICE version 1.3.0
```

Reply:
Ah ok, yes. Still, I would also put the date and that regardless of logger configuration. So that the first preCICE thing you see, when looking as a look is also a time stamp of the run.

Reply:
I changed the order of statements. About the date, I am not so sure. What use case would that be for? Wouldn't that be the job of the solver?

Reply:
Mhh, don't think it can clearly by said it's either the job of the solver or the precice. Anyway, it's fine for me now.